site/layouts/assessment_wrote_en-en.hbs

```diff
@@ -0,0 +1,202 @@
+			</li>
+		</ul>
+		<details>
+			<summary>Subpart of the subject and expertise field involvment</summary>
+			<p>List of subparts that might be assessed independently of the main identified subject. For example: a menu, an image, a video... Note: the following list may be empty if not applicable.</p>
+			<ul data-wb-json='{
+					"url": "#[a11y]/earl:subject/dct:hasPart",
+					"mapping": [
+						{ "selector": "[data-about]", "value": "/dct:description" },
+						{ "selector": "[data-css]", "value": "/earl:pointer/@value" }
+					]
+				}'>
+				<template>
+					<li><span data-about></span> (CSS selector: <code data-css>Not applicable</code>)</li>
+				</template>
+			</ul>
+			<p>Suggested list of expertise topics that might need to be involved in the subject assessment in order to produce a complete accessibility conformance report. Note: the following list may be empty if not applicable.</p>
+			<ul data-wb-json='{
+					"url": "#[a11y]/acr:involvesExpertise",
+					"mapping": { "selector": "[data-expertise]" }
+				}'>
+				<template>
+					<li><span data-expertise></span></li>
+				</template>
+			</ul>
+		</details>
+	</dd>
+	<dt>Toward a conformity at:</dt>
+	<dd><span data-json-replace="#[a11y]/acr:standard">Not specified</span> (Option: <span data-json-replace="#[a11y]/acr:conformanceOption">Not specified</span>)</dd>
+	<dt>Summary:</dt>
+	<dd data-json-replace="#[a11y]/dct:description">No note</dd>
+</dl>
+
+<h2 id="acr">Assessment details</h2>
+<table class="table">
+	<thead>
+		<tr>
+			<th class="col-md-3">Success Criteria</th>
+			<th class="col-md-2">Result</th>
+			<th>Comment</th>
+		</tr>
+	</thead>
+	<tbody data-wb-json='{
+		"url": "#[a11y]/earl:result",
+		"tobeclone": "tr",
+		"source": "#tmpl-result",
+		"mapping": [
+			{ "selector": "[data-label]", "value": "/earl:test" },
+			{ "selector": "[data-outcome]", "value": "/earl:outcome" },
+			{ "selector": "[data-applicability]", "value": "/earl:subject/dct:description" },
+			{ "selector": "[data-notes]", "value": "/dct:description" },
+			{ "selector": "[data-mode]", "value": "/earl:mode" },
+			{ "selector": "[data-severity]", "value": "/acr:severity" },
+			{ "selector": "[data-relevancy]", "value": "/acr:relevancy" },
+			{
+				"selector": "[data-specific-results]",
+				"value": "/earl:result",
+				"mapping": [
+					{ "selector": "[data-result-label]", "value": "/earl:test" },
+					{ "selector": "[data-result-outcome]", "value": "/earl:outcome" },
+					{ "selector": "[data-result-severity]", "value": "/acr:severity" },
+					{ "selector": "[data-result-relevancy]", "value": "/acr:relevancy" },
+					{ "selector": "[data-result-notes", "value": "/dct:description" }
+				]
+			}
+		]
+	}'>
+	</tbody>
+</table>
+
+<template id="tmpl-result">
+	<table>
+		<tr>
+			<th data-label></th>
+			<td><span data-outcome></span></td>
+			<td>
+				<p><strong>Applicability</strong>: <span data-applicability>Not re-specified</span></p>
```

Comment:
"Not re-specified" may be self-explanatory, but I personally don't know what it means. Maybe having a legend somewhere in the future could be useful.

Reply:
This is not mandatory to merge this PR, can be done at a later time.

Reply:
What about "Same as the main subject" ?

This properties is to allow to perform targeted test. For example if the subject is the whole page, then the assessor might want to assess the site menu component separately from the site footer and to the main content of the page.

Reply:
@duboisp Would it be "Scope" then?

Reply:
Yes it is a refinement of the scope that is being assessed.

I like "as defined".

Reply:
fixed